docs/using-the-jdbc-driver/sample-code/HikariSample.java

```diff
@@ -49,7 +50,8 @@ public static void main(String[] args) throws SQLException {
     targetDataSourceProps.setProperty("serverPropertyName", "db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com");
     targetDataSourceProps.setProperty("databasePropertyName", "postgres");
     targetDataSourceProps.setProperty("portPropertyName", "5432");
```

Comment:
These need to be changed as well
```suggestion
    targetDataSourceProps.setProperty("serverName", "db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com");
    targetDataSourceProps.setProperty("databaseName", "postgres");
    targetDataSourceProps.setProperty("portNumber", "5432");
```

Similarly the code above from lines 41-43 should be changed to:
```
    ds.addDataSourceProperty("databasePropertyName", "databaseName");
    ds.addDataSourceProperty("portPropertyName", "portNumber");
    ds.addDataSourceProperty("serverPropertyName", "serverName");
```